src/ui/README.md

```diff
@@ -4,14 +4,61 @@ The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
 package, you can start the UI by running `aqueduct ui`.
```

Comment:
This `aqueduct ui` is also deprecated. Could you update this to "The UI is started together with the backend server via `aqueduct start`"?

Reply:
sure can